client/src/components/MultiJurisdictionAudit/index.tsx

```diff
@@ -111,6 +142,9 @@ export const AuditAdminView: React.FC = () => {
               auditSettings={auditSettings}
             />
           </Inner>
+          <RefreshStatusWrapper>
+            <Tag>{refreshStatus}</Tag>
+          </RefreshStatusWrapper>
```

Comment:
Since the progress table can be arbitrarily long (depending on the number of jurisdictions), I think this might go better at the top of the page so it is always visible. Here are a few places I think it might work well

<img width="1045" alt="Screen Shot 2020-09-17 at 2 50 18 PM" src="https://user-images.githubusercontent.com/530106/93532127-3e0df700-f8f5-11ea-9153-65c472153d0e.png">

Reply:
@jonahkagan I think the best option would be to modify the status box to accept children and display them below the regular content. Putting it in the right would conflict with the round launch buttons and so forth. I'll look into that.